### PR TITLE
Format EditDistance.py

### PR DESCRIPTION
**Pull request by kartikp36**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Request to format my merged file [EditDistance.py](https://github.com/TesseractCoding/NeoAlgo/blob/master/Python/dp/EditDistance.py) according to python style guides
- 4 spaces
- snake_case for variable names
- add driver code

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Improvement on #1537

### CODE REVIEW

File: Python/dp/EditDistance.py [hunk 1]
```diff
@@ -7,49 +7,53 @@
...
+                )
+    return distance[first_length][second_length]
+
+
+if __name__ == '__main__':
+    print("Enter First string: ")
+    first_string = input()
+    print("Enter Second string: ")
+    second_string = input()
+    print(editDist(first_string, second_string))
```
Comment (atarax665) [inline Python/dp/EditDistance.py]: Format the output in more readable form. i.e: ```Minimum number of edits required: x```
Also edit the Sample Output accordingly.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Please add a small of description of the problem and problem link from any competitive coding website.

File: Python/dp/EditDistance.py (hunk 1 above)
Comment (kartikp36) [reply Python/dp/EditDistance.py]: @atarax665 I've made the changes, please review.

**Review by atarax665 — APPROVED** [review]
LGTM :+1:

**Pull request merged**